src/utils/browser/safeLocalStorage.ts

```diff
@@ -0,0 +1,67 @@
+export const safeLocalStorage = {
+  getItem(key: string): string | null {
+    try {
+      if (typeof window !== 'undefined' && window.localStorage) {
+        return window.localStorage.getItem(key)
+      }
+      return null
+    } catch (error) {
+      console.warn('LocalStorage is not accessible:', error)
+      return null
+    }
```

Comment:
You can wrap all try catch in a private method

Reply:
and voilà !